src/lib/php/common-sysconfig.php

```diff
@@ -557,6 +557,19 @@ function Populate_sysconfig()
   $desc = _('');
   $valueArray[$variable] = array("'$variable'", "''", "'$prompt'",
     strval(CONFIG_TYPE_PASSWORD), "'SWH'", "5", "'$desc'", "null", "null");
+  $variable = "ScAPIURL";
+  $prompt = _('Scanoss API url');
+  $desc = _('Set URL to SCANOSS API (blank for default osskb.org');
```

Comment:
A cosmetic change required.
```suggestion
  $desc = _('Set URL to SCANOSS API (blank for default osskb.org)');
```